pkg/v1/tkg/tkgconfigbom/bom.go

```diff
@@ -196,6 +196,7 @@ func (c *client) loadBOMConfigurationFromFiledata(data []byte) (*BOMConfiguratio
 		bomConfiguration.ProvidersVersionMap["cluster-api"] = getSimpleVersion(bomConfiguration.Components["cluster_api"][0].Version)
 		bomConfiguration.ProvidersVersionMap["bootstrap-kubeadm"] = getSimpleVersion(bomConfiguration.Components["cluster_api"][0].Version)
 		bomConfiguration.ProvidersVersionMap["control-plane-kubeadm"] = getSimpleVersion(bomConfiguration.Components["cluster_api"][0].Version)
+		bomConfiguration.ProvidersVersionMap["infrastructure-docker"] = getSimpleVersion(bomConfiguration.Components["cluster_api"][0].Version)
```

Comment:
Can we note in the PR description that we're adding this and why?

Also, is there a corresponding test change?

Reply:
Updated the PR description and added a unit test for it.